testapi.pm

```diff
@@ -390,8 +390,9 @@ sub upload_logs {
     my ($file) = @_;
 
     bmwqemu::log_call('upload_logs', file => $file);
-    my $cmd      = "curl --form upload=\@$file ";
     my $basename = basename($file);
+    my $upname   = ref($autotest::current_test) . '-' . $basename;
```

Comment:
See https://github.com/os-autoinst/os-autoinst/pull/409#issuecomment-177817938 i.e. 'shutdown' we have `shutdown/shutdown.pm` and `x11/shutdown.pm`.